tests/Feature/InstallTest.php

```diff
@@ -125,7 +125,7 @@ public function testInstall(): void
 		/**
 		 * apply migration.
 		 */
-		$response = $this->get('install/migrate');
+		$response = $this->post('install/migrate', ['username' => 'admin', 'password' => 'password']);
 		$this->assertOk($response);
 		$response->assertViewIs('install.migrate');
```

Comment:
Assertion needed : that there exists an admin user at this point and that we can log to it with `admin` and `password.

Reply:
Yes. That is what I would like this assertion to check. that the DB actually do contains an admin user with the said username and password. :)